css-four-column-layout/styles.css

```diff
@@ -0,0 +1,49 @@
+section {
+  font-family: barlow, sans-serif;
+  width: 100%;
+}
+
+body {
+  background-color: rgb(236 236 236);
+}
+
+h2 {
+  font-size: 35px;
+  font-weight: bold;
+  text-align: center;
+}
+
+.img-screen {
+  width: 280px;
+}
+
+.img-icon {
+  width: 75px;
+  margin-bottom: -75px;
+  margin-left: -175px;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  margin-right: 35px;
+}
+
+.column-one-fourth {
```

Comment:
always make sure the column classes have a set width on them, and that set width should be a percentage value